app/client/src/config/mapServiceConfig.js

```diff
@@ -20,6 +20,8 @@ export const wsio = `${watersGeoBase}r4/wsio/MapServer/0`;
 
 export const tribal = `${geopubBase}EMEF/tribal/MapServer`;
 
+export const congressional = `${geopubBase}NEPAssist/Boundaries/MapServer`;
+
```

Comment:
Can you add the congressional layer web service to the mapServiceMapping list below? This will prevent it from showing up as UNKNOWN in Google Analytics.

Should look something like this: 
```
  {
    wildcardUrl: `${congressional}*`,
    name: 'geopub - congressional districts layer',
  },
```

Reply:
@cschwinderg added the Congressional service to the service mapping list, thanks!